bower.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "asq-option",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "homepage": "https://github.com/ASQ-USI-Elements/asq-option",
   "authors": [
     "Vasileios Triglianos <triglian@gmail.com>",
-    "Zhenfei Nie <youyis2fox@gmail.com>"
+    "Zhenfei Nie <youyis2fox@gmail.com>",
+    "Gustavo Graziani <gustavo.graz@outlook.com>"
   ],
-  "description": "ASQ option element. To be used with asq-multi-choice-q question elements.",
+  "description": "ASQ option element with optional text input for other options. To be used with asq-multi-choice-q question elements.",
```

Comment:
ASQ option element with optional text input for other options -> ASQ option element with optional text input when the option is configured as an "other" option .